tests/e2e/verfication_post_day1_test.go

```diff
@@ -176,8 +176,9 @@ var _ = Describe("TF Test", func() {
 					err = importService.Import(importParam)
 					Expect(err.Error()).To(ContainSubstring("Cannot import non-existent remote object"))
 
-					// skip due to bug :: OCM-5246
-					// Expect(output).To(ContainSubstring(profile.ComputeMachineType))
+					By("remove .tfstate file to prevent destructive outcomes")
+					defer h.DeleteFile(con.GrantTFstateFile(con.ImportResourceDir))
```

Comment:
May be add function to the Service with the clean up. @xueli181114  WDYT?

Reply:
You are right. But may be add function to utils somewhere, where we will give manifest folder to clean up the state?

Reply:
ack, I added a function :D